RELEASES.md

```diff
@@ -1,3 +1,133 @@
+Version 1.62.0 (2022-06-30)
+==========================
+
+Language
+--------
+
+- [Stabilize `#[derive(Default)]` on enums with a `#[default]` variant][94457]
+- [Stop validating some checks in dead code after functions with uninhabited return types][93313]
+- [Fix constants not getting dropped if part of a diverging expression][94775]
+- [Support unit struct/enum variant in destructuring assignment][95380]
+- [Allow inline consts to reference generic params][96557]
```

Comment:
Inline consts are nightly-only. I presume nightly-only things usually should not be included in the release notes?

Reply:
You're correct, removed the line.